numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -1740,6 +1740,8 @@ def add_newdoc(place, name, doc):
     """
     Test element-wise for NaT (not a time) and return result as a boolean array.
 
+    .. versionadded:: 1.13.0
+
```

Comment:
Should be preceded by a Notes heading, see other examples in the file (although in `_adddocs.py` I do see examples that do not use the heading).

Reply:
Most of the more "recent" (1.12 and 1.13) `.. versionadded` were included in the general description.  Also it seemed a bit wasteful to add another section just for the versionadded. But if you like it better in the `Notes` I can change that.

Reply:
We might want to revisit that. I think it makes more sense to put the `versionadded` in the extended summary where it is easily noticed, likewise, when arguments are added we put the `versionadded` with the argument documentation. @rgommers Thoughts?

Reply:
What about leaving the possible unification to a separate issue (and/or pull request) and merge this PR since it's (probably) an improvement? 😄

Reply:
@eric-wieser Yep, I think that is fine and prefer it that way. However, the documentation for that says to put it in the Notes, which I haven't done myself (tsk, tsk), so we should fix the documentation.